.travis.yml

```diff
@@ -16,10 +16,8 @@ env:
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-          env: DEPLOY=yes TASK_TESTS_COVERAGE=1
         - php: 7.1
-          env: TASK_SCA=1
```

Comment:
it's enough to run SCA only on `nightly`, as we treat it as non-experimental job